Vagrantfile

```diff
@@ -226,7 +226,7 @@ Vagrant.configure(VAGRANTFILE_API_VERSION) do |config|
       libvirt.memory      = vagrant_openshift_config['memory'].to_i
       libvirt.cpus        = vagrant_openshift_config['cpus'].to_i
       # run on libvirt somewhere other than default:
-      libvirt.uri         = ENV["VAGRANT_LIBVIRT_URI"] if defined? ENV["VAGRANT_LIBVIRT_URI"] 
+      libvirt.uri         = ENV["VAGRANT_LIBVIRT_URI"] unless defined? ENV["VAGRANT_LIBVIRT_URI"]
```

Comment:
If it was (and still is) working without the `libvirt.uri` line, then I'm assuming it's the default value (the `qemu:///system`). My intention with this change was to set that value to `VAGRANT_LIBVIRT_URI` if one was specified, thus the condition `unless defined?`, otherwise it wouldn't be set (falling back to default).
I've also tried your solution, which works cleanly, now the question is which one you want here?